sorc/ncep_post.fd/WRFPOST.f

```diff
@@ -342,15 +347,15 @@ PROGRAM WRFPOST
           if(me == 0) then
             print*,'using pressure levels from POSTGPVARS'
           endif
-          if(MODELNAME == 'RAPR')then
-            read(5,*) (po(l),l=1,kpo)
+!KaYee          if(MODELNAME == 'RAPR')then
+!KaYee            read(5,*) (po(l),l=1,kpo)
 ! CRA READ VALID TIME UNITS
-            read(5,121) VTIMEUNITS
-            if(me == 0) then
-              print*,'VALID TIME UNITS = ', VTIMEUNITS
-            endif
+!KaYee            read(5,121) VTIMEUNITS
```

Comment:
@kayeekayee I would like to keep reading vtimeunits as a option. Could add the change in line 176 as:
namelist/nampgb/kpo,po,kth,th,kpv,pv,fileNameAER,d3d_on,gocart_on,popascal &
                     ,hyb_sigp,rdaod,aqfcmaq_on,vtimeunits